### PR TITLE
Feature/5573 develop server

### PR DESCRIPTION
**Pull request by jesuspg**
@flopezag 
@jframos

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,32 @@
+===============
+ FIWARE aiakos
```
Comment (flopezag) [inline README.rst]: Aiakos

File: lib/public/keys/spain.gpgkey [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+XPdXAIAAku46hqvTdcQRImd4CJIP2+XvHKlMMsU5jmAR/jpvFbq3JdlJ2QlHTBCS
+mn7Gd5HdZtRQ960AEQEAAYkBHwQYAQIACQUCVadUJgIbDAAKCRCmBOB+bCRs1F4y
+B/98XV+mi0yWAlCNZjyiXzYDtU7NKAyluHnVaxsGk8b5rf7Ysm16sWeEK6iUqKMn
+F/UxTBS0eex0Ycb7JapEd/hLuT/6gHnIR1tJkSXkR8JHyjDslFQqePoDuqhMtWHv
+4duS5/rM2hJwavvtr1pVymiQ05e/HlZEH6M3jb6V8FCHDg6aHNph97vG7GNaxi00
+mX7DWOCYc+0GUlTTMDnVpypgowAW5ExY/EFENvfcCCJTNklm1HoKCQ1UN2wAxtlu
+jubEr3vxrj3em0CnkfKSczzbdM55MSJ/D+kkjrwvpVva69urbLrg+kzljtSMMbMb
+nMy+mcDYEPbM6AXnATo4EFIm
+=TrvV
+-----END PGP PUBLIC KEY BLOCK-----
```
Comment (flopezag) [inline lib/public/keys/spain.gpgkey]: add new line?

File: lib/routes/v1.js [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+module.exports.validateRegion = validateRegion;
+
+/** @export */
+module.exports.validateKey = validateKey;
+
+/** @export */
+module.exports.getKey = getKey;
+
+/** @export */
+module.exports.setKeysPath = setKeysPath;
```
Comment (flopezag) [inline lib/routes/v1.js]: add new line

File: test/unit/region1.gpgkey [hunk 4]
```diff
@@ -0,0 +1,5 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
+Version: GnuPG v1
+
+mQENBFWnVCYBCADPeDMbTOkCM4MPbUMvtbAtGbUDnH3AHyZCEZZuyjeExATfT0Au
+-----END PGP PUBLIC KEY BLOCK-----
```
Comment (flopezag) [inline test/unit/region1.gpgkey]: add new line?

File: test/unit/test_v1.js [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+        res.end = sinon.spy();
+
+        v1.getKey(req, res, next);
+
+        assert(res.type.calledOnce);
+        assert(res.end.calledOnce);
+
+    });
+
+});
```
Comment (flopezag) [inline test/unit/test_v1.js]: add new line

File: README.rst [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+===============
+
+|Build Status| |Coverage Status|
+
+Server with public API to manage ssh/gpg keys for the support user of each
```
Comment (flopezag) [inline README.rst]: ssh/gpg public keys

File: lib/routes/v1.js (hunk 3 above)
Comment (jframos) [inline lib/routes/v1.js]: No newline at the end of the file.

File: test/unit/test_v1.js (hunk 5 above)
Comment (jframos) [inline test/unit/test_v1.js]: No newline.

**Comment by jframos** [comment]
Are we going to upload the public keys to this repo?

**Comment by jframos** [comment]
I do not know if it is a good idea adding a new empty line at the end of public ssh-key files.

**Comment by jframos** [comment]
_README.rst_ should contains a brief outline about the API with some curl examples.
Are we going to create Apiary documentation for this componente?

File: Dockerfile [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+FROM centos
+WORKDIR /opt
+RUN yum -y install git
+RUN curl --silent --location https://rpm.nodesource.com/setup | bash -
+RUN yum -y install nodejs
+RUN yum -y install rpm-build
+RUN yum -y install initscripts
+RUN git clone https://github.com/telefonicaid/fiware-aiakos fiware-aiakos-source
+WORKDIR /opt/fiware-aiakos-source
+RUN git checkout feature/5573_develop_server
```
Comment (jframos) [inline Dockerfile]: We should work with stable code; master branch?

**Comment by jframos** [comment]
LGTM

@flopezag ?

**Comment by flopezag** [comment]
LGTM

**Pull request merged**